detox/test/src/Screens/WebViewScreen.js

```diff
@@ -119,7 +119,7 @@ const webViewFormWithScrolling = `
         </style>
     </head>
     <body>
-        <h1 id="pageHeadline">First Webview</h1>
+        <h1 id="pageHeadline" aria-label="first-webview">First Webview</h1>
```

Comment:
So, do we have secured webview in the test here? If not, how do we gain confidence in the API??

Reply:
I have tests written in a different branch (`temp-secured-webview-2`), our test app needs some changes to make them work because currently on iOS, things aren't accessible* with XCUITest. 
*we have that overlay that blocks any accessibility tool. Just check it with the Inspector, and you'll see that you can't tap on elements. Similarly, XCUITest is unable to see those elements. This will require a change to the test app, I already solved this problem before and plan to push it here or in a separated change)
Regardless our tests, I also tested it with the Payments module, which also gives us high confidence (probably the most important)

Reply:
added.